packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.controller.js

```diff
@@ -0,0 +1,73 @@
+import get from 'lodash/get';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $anchorScroll,
+    $location,
+    $translate,
+    Alerter,
+    datacenterBackupService,
+  ) {
+    this.$anchorScroll = $anchorScroll;
+    this.$location = $location;
+    this.$translate = $translate;
+    this.Alerter = Alerter;
+    this.backup = {};
```

Comment:
This is usually done in the `$onInit` hook

Reply:
This is not required and has been removed